nautobot/dcim/models/devices.py

```diff
@@ -544,6 +545,20 @@ class Device(PrimaryModel, ConfigContextModel, StatusModel):
         blank=True,
         null=True,
     )
+    redundancy_group = models.ForeignKey(
```

Comment:
I think we should allow for the device to be a member of multiple redundancy groups/domains.

Reply:
@lampwins?

Is this a known need at this time?

Reply:
Un-resolving - Sorry I missed the answer written in invisible font during moonlight. :-)